src/chrome/content/rules/Deutsche-Rentenversicherung.de.xml

```diff
@@ -0,0 +1,26 @@
+<ruleset name="Deutsche-Rentenversicherung.de">
+
+	<target host=       "deutsche-rentenversicherung.de" />
+	<target host=   "www.deutsche-rentenversicherung.de" />
+	<target host="static.deutsche-rentenversicherung-bund.de" />
+	<target host=   "www.deutsche-rentenversicherung-bund.de" />
+	<target host=       "deutsche-rentenversicherung-bund.de" />
+	<target host="www.drv-bund.de" />
+	<target host=    "drv-bund.de" />
+
+  	<securecookie host="^www\.deutsche-rentenversicherung\.de$" name=".+" />
+
+	<!-- could not resolve host -->
+	<rule from="^http://deutsche-rentenversicherung(-bund)?\.de/"
```

Comment:
http://www.deutsche-rentenversicherung-bund.de http://deutsche-rentenversicherung-bund.de https://www.deutsche-rentenversicherung-bund.de are redirected (by server) to http://www.deutsche-rentenversicherung.de just fine, except for https://deutsche-rentenversicherung.de/ & https://deutsche-rentenversicherung-bund.de/ so line 14 is necessary.